resources/core/charts/monitoring/values.yaml

```diff
@@ -2,5 +2,5 @@
 
 test:
   image:
-    name: test-logging-monitoring
-    tag: 0.2.37
+    name: snapshot/test-logging-monitoring
```

Comment:
why do you need to update monitoring

Reply:
The orchestrator detected a change and rebuilt it so it will need a version bump too.

Reply:
Ok, reverted.